.github/workflows/package-component.yaml

```diff
@@ -4,7 +4,11 @@ on:
   push:
     tags:
       # Matches: new-components/scanners/gosec/v1.0.0 or new-components/scanners/gosec/v1.0.0-alpha3 for example.
-      - 'new-components/(scanners|targets|reporters|filters|enrichers)/[^/]+/v[0-9]+\.[0-9]+\.[0-9]+.*'
+      - 'new-components/targets/*/v[0-9].[0-9].[0-9]*'
```

Comment:
This is interesting but `act` was succeeding locally while GH actions complained about it on CI as they actually don't fully support regex